Assets/MRTK/SDK/Features/UX/Scripts/InteractiveElement/BaseInteractiveElement.cs

```diff
@@ -142,21 +178,48 @@ public void OnTouchCompleted(HandTrackingInputEventData eventData)
 
         public void OnTouchUpdated(HandTrackingInputEventData eventData)
         {
-            if (IsStatePresent(TouchStateName))
-            {
-                EventReceiverManager.InvokeStateEvent(TouchStateName, eventData);
-            }
+            EventReceiverManager.InvokeStateEvent(TouchStateName, eventData);
         }
 
         #endregion
 
+        #region SelectFar
+
+        public void OnPointerDown(MixedRealityPointerEventData eventData)
+        {
+            SetStateAndInvokeEvent(SelectFarStateName, 1, eventData);
+        }
+
+        public void OnPointerDragged(MixedRealityPointerEventData eventData)
+        {
+            EventReceiverManager.InvokeStateEvent(SelectFarStateName, eventData);
+        }
+
+        public void OnPointerClicked(MixedRealityPointerEventData eventData)
+        {
+            EventReceiverManager.InvokeStateEvent(SelectFarStateName, eventData);
+
+            TriggerClickedState();
+
+            SetToggleStates();
+        }
+
```

Comment:
By default, the Clicked state and the ToggleOn/Off states are set by the `IMixedRealityPointerHandler`'s far interaction OnPointerClicked.

If a user wants to trigger a click or set the toggle states from another entry point such as OnTouchStarted, they can use the `TriggerClickedState()` and `SetToggleStates()` methods.

Reply:
Where are we ensuring that people know about this trick? I would recommend via an example prefab, and in documentation (e.g. via sample code).

Reply:
I plan on having an example scene and multiple example prefabs, along with documentation scenarios.